src/Elmish.WPF/ViewModel.fs

```diff
@@ -196,21 +187,22 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
     | CmdData d ->
         let exec = measure name "exec" d.Exec
         let canExec = measure name "canExec" d.CanExec
-        let execute _ = exec currentModel |> ValueOption.iter dispatch
-        let canExecute _ = canExec currentModel
+        let execute _ = currentModel |> exec |> ValueOption.iter dispatch
+        let canExecute _ = currentModel |> canExec
         Cmd (Command(execute, canExecute, false), canExec)
     | CmdParamData d ->
         let exec = measure2 name "exec" d.Exec
         let canExec = measure2 name "canExec" d.CanExec
-        let execute param = exec param currentModel |> ValueOption.iter dispatch
-        let canExecute param = canExec param currentModel
+        let execute param = currentModel |> exec param |> ValueOption.iter dispatch
+        let canExecute param = currentModel |> canExec param
         CmdParam <| Command(execute, canExecute, d.AutoRequery)
     | SubModelData d ->
         let getModel = measure name "getSubModel" d.GetModel
         let getBindings = measure name "bindings" d.GetBindings
         let toMsg = measure name "toMsg" d.ToMsg
         match getModel initialModel with
-        | ValueNone -> SubModel (ref ValueNone, getModel, getBindings, toMsg, d.Sticky)
```

Comment:
I like this better as it currently is. I generally put short lines on the same line. (I do make some exceptions, for example if there are many cases and most of them have their own lines, or sometimes if I match against multiple cases.)

Reply:
I prefer consistency in parallel structure over fewer lines.  If you insist though, I will change it back.